<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Annonce</class>
 <widget class="QDialog" name="Annonce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {background-image: url('../Ressources/background-dialog.jpg')}</string>
  </property>
  <widget class="QWidget" name="container" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>751</width>
     <height>541</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#container {
	border-radius: 5px;
	background-image: url('../Ressources/background_alpha.png');
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Détails de l'annonce</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Type de bien</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_type">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="rdb_vente">
            <property name="text">
             <string>Vente</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rdb_location">
            <property name="text">
             <string>Location</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Surface habitable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_surface">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Surface en mètres carrés</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Surface du terrain</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_surface_terrain">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>En mètres carrés, si nécéssaire</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Nombre de pièces</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_pieces">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Adresse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_addr1">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Ligne 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_addr2">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Ligne 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_addr3">
          <property name="placeholderText">
           <string>Ligne 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Prix</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_prix">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Saisissez un prix en euros</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txt_desc">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {border-radius: 2px;}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Photo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_img">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>350</height>
           </size>
          </property>
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Image représentant le bien</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_addPohot">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #0275d8;
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
	color: white;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
          </property>
          <property name="text">
           <string>Ajouter une photo depuis mon ordinateur</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_date">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Mise en ligne le :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chkOccupe">
        <property name="text">
         <string>Est occupé</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #d9534f;
    border-style: outset;
    border-radius: 10px;
	border-width: 1px;
	border-color: beige;
	color: white;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #5cb85c;
    border-style: outset;
    border-radius: 10px;
	border-width: 1px;
	border-color: beige;
	color: white;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Sauvegarder</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_save</sender>
   <signal>clicked()</signal>
   <receiver>Annonce</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Annonce</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>540</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_addPohot</sender>
   <signal>clicked()</signal>
   <receiver>Annonce</receiver>
   <slot>openImageFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOccupe</sender>
   <signal>clicked(bool)</signal>
   <receiver>Annonce</receiver>
   <slot>setEstOccupe(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openImageFile()</slot>
  <slot>setEstOccupe(bool)</slot>
 </slots>
</ui>
